libs/orchestration/terraform-azurerm-amido-aks/examples/entire-infra/constraints.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_version = ">= 0.12"
   required_providers {
-    azurerm = "~> 1.44.0"
+    azurerm = "~> 2"
```

Comment:
No point version?

Reply:
let's stick to latest 2.x version for now - it just shouldn't be 1.x